### ISSUE TITLE
Test: confirmation dialog with checkbox for certain explorer operations

### ISSUE THREAD

**Issue by bpasero** [Microsoft/vscode#36990]
- [ ] win
- [ ] os x
- [ ] linux

Refs: https://github.com/Microsoft/vscode/issues/1414 and https://github.com/Microsoft/vscode/issues/1538
Complexity: 2

We added confirmation dialogs with checkboxes to the following actions:
* move a file/folder via DND in the explorer
* delete a file/folder from the explorer

Verify that you see this dialog and that you can set the checkbox so that you are not being asked again. Verify there are related settings to bring back the confirmation.

**Comment by weinand** [comment]
@bpasero why does "move" need a confirmation dialog at all?
I'm not aware of any file explorer that has this.
(This gets especially annoying if there is already a file/folder with the same name in the target location: then you there are two dialogs...)

**Comment by weinand** [comment]
ok, I saw the long discussion about this in #1414

**Comment by bpasero** [comment]
@weinand I think we would maybe not need this dialog if it was harder to do the move operation by accident. I remember our good old **hysteresis** discussion a couple of years ago for RTC around a similar issue ;)

**Comment by weinand** [comment]
Yes, **hysteresis** and **undo** are always an unbeatable combination ;-)
